pkg/util/gziphandler/gzip.go

```diff
@@ -393,15 +391,15 @@ func (c *config) validate() error {
 	return nil
 }
 
```

Comment:
This is a nit, but with option patterns, I really like the `WithMyOption` naming pattern (here `WithCompressionLevel` so that it shows the additive nature of running the func on the config). I figure since we're taking the code anyway, might as well adjust for readability and clarity.

Reply:
I see now that it clarifies by using `GzipHandlerWithOps(opts ...Option)` so this comment is redundant